rest/bulk_api.go

```diff
@@ -195,8 +199,8 @@ func (h *handler) handleAllDocs() error {
 	// Now it's time to actually write the response!
 	lastSeq, _ := h.db.LastSequence()
 	h.setHeader("Content-Type", "application/json")
+	h.writeStatus(http.StatusOK, http.StatusText(http.StatusOK))
```

Comment:
Stick a comment in here explaining why we're setting the status here, even though the request might subsequently fail. (i.e. response.Write is setting implicitly, doc-level errors are handled in the response, broken pipe doesn't require header)